klaas/scripts/train_separation_model.py

```diff
@@ -50,14 +50,17 @@ def main(configuration_path, signal_path, background_path, predictions_path, mod
     n_cross_validations = config.get('n_cross_validations', 10)
 
     training_variables = config['training_variables']
-
+    
     classifier = eval(config['classifier'])
 
     check_extension(predictions_path)
     check_extension(model_path, allowed_extensions=['.pmml', '.pkl'])
 
     log.info('Loading signal data')
-    df_signal = read_data(file_path=signal_path, key=key)
+    df_signal = read_data(file_path=signal_path, key=key,
+                          columns=training_variables.append(
+                              'MCorsikaEvtHeader.fTotalEnergy')
```

Comment:
actually it does not make that much sense here, because here it is not the target, it is an additional information that is used for the performance plot